aiinfra-cluster/main.tf

```diff
@@ -70,9 +70,8 @@ locals {
     }
   ] : []
 
-  widgets = (var.orchestrator_type != "gke" && var.enable_ops_agent) ? [
-    for widget_object in module.dashboard-metric-descriptor.widget_objects : jsonencode(widget_object)
-  ] : []
+  mig_widgets = var.enable_ops_agent ? concat(module.dashboard-metric-descriptor.nvidia_dcgm_widgets, module.dashboard-metric-descriptor.nvidia_nvml_widgets) : []
+  gke_widgets = var.enable_ops_agent ? module.dashboard-metric-descriptor.gke_gpu_utilization_widgets : []
```

Comment:
This is actually both gce and gke.

So `gke_gpu_utilization_widgets` should be renamed to something like `gce_gke_gpu_utilization_widgets`, and it should be in both mig as well as gke.

Reply:
Addressed in latest iteration.